packages/plugins/plugin-deck/src/components/Plank/PlankControls.tsx

```diff
@@ -24,6 +24,8 @@ export type PlankControlHandler = (event: DeckAction.PartAdjustment) => void;
 export type PlankCapabilities = {
   incrementStart?: boolean;
   incrementEnd?: boolean;
+  // TODO(burdon): Variants.
+  deck?: boolean;
```

Comment:
I don't understand this, what is the "deck" capability/action?

Reply:
if not enabled, then it would lock solo mode.